From 910b1a1cfb56b1476ce49a0c1cac312c4a202ab4 Mon Sep 17 00:00:00 2001
From: Jiyong Park <jiyong@google.com>
Date: Mon, 7 Dec 2020 15:08:36 +0900
Subject: [PATCH] add crosvm to com.android.virt

To do so, crosvm and its dependencies have the apex_available property
set to "//apex_available:platform", "com.android.virt" to explicitly
acknowledge the joining.

Bug: 174639241
Test: m com.android.virt

Change-Id: Idd7706a931fde0dfd77fe8b58f6f0c84081c6e3c
---
 Android.bp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Android.bp b/Android.bp
index 8b264cd3..17b079d9 100644
--- a/Android.bp
+++ b/Android.bp
@@ -66,4 +66,8 @@ cc_library {
         "-Wno-sign-compare",
         "-Wno-tautological-compare",
     ],
+    apex_available: [
+        "//apex_available:platform",
+        "com.android.virt",
+    ],
 }
-- 
2.17.1

